docker-compose.yml

```diff
@@ -16,13 +16,15 @@ services:
   mongo:
     image: mongo
     restart: always
+    env_file:
+      - .env
     ports:
       - "27017:27017"
     environment:
       MONGO_INITDB_ROOT_USERNAME: root
       MONGO_INITDB_ROOT_PASSWORD: toor
     volumes:
-      - mongo-data-dev:/data/db     #change into host volume
+      - ${MONGO_DATA_PATH}:/data/db
```

Comment:
Would be nice to have a docker-compose file for dev environment too.

Reply:
True. I will add that :D